ProjectLighthouse.Servers.Website/Controllers/AutoDiscoverController.cs

```diff
@@ -0,0 +1,26 @@
+using LBPUnion.ProjectLighthouse.Configuration;
+using LBPUnion.ProjectLighthouse.Servers.Website.Types;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LBPUnion.ProjectLighthouse.Servers.Website.Controllers;
+
+[ApiController]
+public class AutoDiscoverController: ControllerBase
+{
+    [ResponseCache(Duration = 86400)]
```

Comment:
Did you test if this actually does anything? I think in ASP.NET you have to explicitly enable response caching at startup.

Reply:
<img width="658" alt="image" src="https://github.com/user-attachments/assets/215cf52f-fbd5-481d-b0f0-a1ccc003a6aa" />
yep, it works fine